apps/common/controllers/add-another-address-loop.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const _ = require('lodash');
+const BaseController = require('./base');
+
+module.exports = class AddAnotherAddressController extends BaseController {
+  locals(req, res, callback) {
+    const locals = super.locals(req, res, callback);
+    const field = locals.field;
+    const addresses = req.sessionModel.get(`${field}Addresses`);
+    const hasAddresses = req.sessionModel.get(`${field}Addresses`) ? true : false;
+    const items = [];
+    _.forEach(addresses, (value, key) => {
+      const address = {
+        id: key,
+        address: value.address
+      };
+      items.push(address);
+    });
```

Comment:
Firstly, we use `_.each` in quite a few places in the codebase - I would prefer us to keep using `_.each` rather than an alias.

Secondly - would it not be better to create the items array in one operation?
```js
const items = Object.keys(addresses).map(key => ({
  id: key,
  address: addresses[key].address
});

or

const items = _.map(addresses, (value, key) => ({
  id: key,
  address: value.address
}));
```